dockerfiles/ci/centos/7/base.Dockerfile

```diff
@@ -239,6 +239,25 @@ RUN source scl_source enable devtoolset-7 \
     && cd - \
     && rm -fr "$FILENAME" "${FILENAME%.tar.gz}"
 
+# Install rust-src manually, since it's not included in the offline installer.
+# Levi figured this out through reading the rustup script and trial and error.
+RUN rustver="1.84.1" \
```

Comment:
Can we avoid hardcoding twice here and just use `${RUST_VERSION}` (defined a few lines above)?